### PR TITLE
Fix: custom path for CLI

### PR DESCRIPTION
**Pull request by PeterSchafer**
### Description
remove incorrect fix-up of custom paths

### Checklist

- [x] Tests added and all succeed
- [ ] Linted
- [x] CHANGELOG.md updated
- [ ] README.md updated, if user-facing

### Screenshots / GIFs
N/A

### CODE REVIEW

File: tests/src/test/java/io/snyk/languageserver/LsRuntimeEnvironmentTest.java [hunk 1]
```diff
@@ -145,4 +145,14 @@ void testEnableTelemetryIsAddedToEnvironmentDisabled() throws StorageException {
     // This is a bit confusing - CLI takes DISABLE as env variable, but we ask for ENABLE, so it's reverted
     assertEquals("1", env.get(Preferences.ENABLE_TELEMETRY));
   }
+  @Test
+  void testAddPath() throws StorageException {
+	String expected = "C;/myPath/:";
```
Comment (bastiandoetsch) [inline tests/src/test/java/io/snyk/languageserver/LsRuntimeEnvironmentTest.java:150]: tha'ts a beautiful path.

File: tests/src/test/java/io/snyk/languageserver/LsRuntimeEnvironmentTest.java [hunk 2]
```diff
@@ -145,4 +145,14 @@ void testEnableTelemetryIsAddedToEnvironmentDisabled() throws StorageException {
...
+    HashMap<String, String> env = new HashMap<>();
+    when(preferenceMock.getPref(Preferences.PATH_KEY)).thenReturn(expected);
+    
+    environment.addPath(env);
+
+    assert(env.get("PATH").startsWith(expected));
```
Comment (bastiandoetsch) [inline tests/src/test/java/io/snyk/languageserver/LsRuntimeEnvironmentTest.java:156]: Well, wouldn't the path just be transmitted to LS, and not added to the env?

File: plugin/src/main/java/io/snyk/languageserver/LsRuntimeEnvironment.java [hunk 3]
```diff
@@ -123,16 +123,9 @@ public void addPath(Map<String, String> env) {
...
-        splitBy = ";";
-      } else {
-        splitBy = ":";
-      }
-      String newPath = path.replace(splitBy, File.pathSeparator)
-        + File.pathSeparator
+      String newPath = path + File.pathSeparator
         + System.getenv("PATH");
-      env.put("PATH", newPath);
+		env.put("PATH", newPath);
```
Comment (bastiandoetsch) [inline plugin/src/main/java/io/snyk/languageserver/LsRuntimeEnvironment.java:128]: I wonder if this should be done.

File: tests/src/test/java/io/snyk/languageserver/LsRuntimeEnvironmentTest.java (hunk 2 above)
Comment (PeterSchafer) [reply tests/src/test/java/io/snyk/languageserver/LsRuntimeEnvironmentTest.java:156]: While addPath() still lives at LsRuntimeEnviornment, it is now only used for CLI invocations. This is why it still needs to be fixed.

**Review by bastiandoetsch — APPROVED** [review]

**Pull request merged**